sql/sp_head.h

```diff
@@ -1576,17 +1576,17 @@ class sp_instr_jump : public sp_instr_opt_meta
 
   virtual ~sp_instr_jump() = default;
 
-  virtual int execute(THD *thd, uint *nextp);
+  virtual int execute(THD *thd, uint *nextp) override;
 
-  virtual void print(String *str);
+  virtual void print(String *str) override;
 
-  virtual uint opt_mark(sp_head *sp, List<sp_instr> *leads);
+  virtual uint opt_mark(sp_head *sp, List<sp_instr> *leads) override;
 
-  virtual uint opt_shortcut_jump(sp_head *sp, sp_instr *start);
+  virtual uint opt_shortcut_jump(sp_head *sp, sp_instr *start) override;
 
-  virtual void opt_move(uint dst, List<sp_instr> *ibp);
+  virtual void opt_move(uint dst, List<sp_instr> *ibp) override;
 
-  virtual void backpatch(uint dest, sp_pcontext *dst_ctx)
+  virtual void backpatch(uint dest, sp_pcontext *dst_ctx) override
   {
     /* Calling backpatch twice is a logic flaw in jump resolution. */
     DBUG_ASSERT(m_dest == 0);
```

Comment:
`dst_ctx` is an unused parameter and should be omitted. Also the `virtual` is redundant because it is already implied by `override`.